### ISSUE TITLE
Put logos on landing page

### ISSUE THREAD

**Issue by edwardchalstrey1** [colouring-cities/colouring-london#102]

**Comment by polly64** [comment]
@mdsimpson42 yes I have to sort this- we have quite a lot of good other ones but I have to recheck ok to use .

**Comment by polly64** [comment]
for moment can you just also add Turing and UCL, OS , GLA and Charnwood council, EPSRC, Geospatial Commission, Innovate UK - do you have these?

**Comment by mdsimpson42** [comment]
We already have Turing, UCL, OS, Historic England, Mayor of London (+Loughborough and Newcastle).

The ones we are missing are:
- [ ] GLA (if different from Mayor of London?)
- [ ] Charnwood council, 
- [ ] EPSRC, 
- [ ] Geospatial Commission, 
- [ ] Innovate UK

I don't think we have them anywhere. I can Google them, but some organisations are prescriptive about which versions of the logo you should use, so it would be better if we got them from an official press pack or something.

**Comment by polly64** [comment]
@mdsimpson42 don't worry I'll sort thanks
